src/uct/api/uct.h

```diff
@@ -896,8 +896,9 @@ struct uct_md_attr {
         size_t               max_alloc; /**< Maximal allocation size */
         size_t               max_reg;   /**< Maximal registration size */
         uint64_t             flags;     /**< UCT_MD_FLAG_xx */
-        uint64_t             reg_mem_types; /** UCS_BIT(uct_memory_type_t) */
-        uct_memory_type_t    mem_type;  /**< Supported(owned) memory type */
+        uint64_t             reg_mem_types; /**< Registered memory type bitmap >*/
+        uint64_t             detect_mem_types; /**< Detect memory type bitmap */
+        uct_memory_type_t    access_mem_type; /**< Access memory type >*/
```

Comment:
Can you please elaborate on each one of the fields a bit more. The only extra info in the comment is "bitmap" which does not provide much info

Reply:
Please see if modified description is better